components/mixins/experience-event/industry-verticals/experienceevent-dining-reservation.schema.json

```diff
@@ -19,100 +19,191 @@
   "definitions": {
     "reservations": {
       "properties": {
-        "xdm:reservations": {
+        "xdm:diningReservation": {
```

Comment:
Still missing the "reservations" object here.

Reply:
Fixed